### DISCUSSION TITLE
Using timestamps for /osm-users/ids

### DISCUSSION THREAD

**Discussion by ramyaragupathy** [hotosm/osm-stats-api#129] [Ideas]
/osm-users/ids endpoint takes from and to timestamps in the request body. The purpose of the endpoint is to retrieve OSM user id for a given OSM username. So it will be good to discard the timestamp inputs here, that way we can use it across different pages that we built for the frontend.

cc @itskshitiz321 @omranlm @robsavoye

**Comment by kshitijrajsharma** [comment]
For the efficient performance  currently we are taking timestamp as parameter to extract it from osm_changeset table.We can solve this issue by tracking users  in users table (up to date ) with available sources ! So that we don't have to extract user statistics from changesets everytime !
